scripts/update_snapshots.sh

```diff
@@ -6,16 +6,33 @@ set -eu
 toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
 cd "${toplevel}"
 
-yarn backend
-echo "Updating GitHub graphql flow types"
-node ./bin/generateGithubGraphqlFlowTypes.js > src/plugins/github/graphqlTypes.js
-echo "Updating for sharness/test_load_example_github.t"
+tmpdir="$(mktemp -d)"
+cleanup() {
+    rm -r "${tmpdir}"
+}
+trap cleanup EXIT
+
+SOURCECRED_BIN="${tmpdir}/bin"
+yarn run --silent backend --output-path "${SOURCECRED_BIN}"
+export SOURCECRED_BIN  # for Sharness and shell tests
+export NODE_PATH="${toplevel}/node_modules${NODE_PATH:+:${NODE_PATH}}"
+
+echo "Updating GitHub GraphQL Flow types"
+cp .prettierrc.json "${SOURCECRED_BIN}/"
```

Comment:
Why is this needed?

Reply:
The Flow types are formatted according to the active Prettier config. If
this is omitted, then the output file will omit trailing commas (etc.),
and so it will not actually be the right snapshot.

Reply:
Got it. It would be good to have a comment explaining in the source code, since it's quite non-obvious.